kpi/models/asset.py

```diff
@@ -666,7 +666,8 @@ def get_filters_for_partial_perm(self, user_id, perm=PERM_VIEW_SUBMISSIONS):
         return None
 
     def get_grant_permissions(self):
-        return self.permissions.filter(deny=False)
+        return self.permissions.filter(deny=False).\
```

Comment:
Let's move this method and `get_label_for_permission()` into `ObjectPermissionMixin`

Reply:
Let's remove it altogether! :boom: